src/domUpdates.js

```diff
@@ -74,14 +77,27 @@ const domUpdates = {
                 <li id="bidet">${room.bidet}</li>
                 <li id="numBeds">${room.numBeds}</li>
                 <li id="cost">${room.costPerNight}</li>
-                <button>Book</button>
+                <button class="book-button" id=${room.number}>Book</button>
             </article>`;
             });
         };
     },
 
-    confirmBooking() {
-    console.log("im confirming");
+    confirmBooking(event, rooms) {
+        const today = new Date().toISOString().split("T")[0];
+        const roomBook = rooms.find(room => event.target.id == room.number);
+        const booking = {
+            "userId": currentUser.id,
+            "date": today,
+            "roomNumber": roomBook.number
+        }
+        fetch("http://localhost:3001/api/v1/bookings", {
```

Comment:
This looks right to me! Only follow up would be how are you updating the DOM with the new data?

Reply:
This is exactly where I left it last night, and I had horrible coding dreams about this exact issue. Todays project I suppose.